api/stripe.go

```diff
@@ -84,6 +86,120 @@ func (a *API) handleWebhook(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 		log.Debugf("stripe webhook: subscription %s for organization %s processed successfully", subscription.ID, org.Address)
+	case "customer.subscription.updated", "customer.subscription.deleted":
```

Comment:
why are both events handled in the same case and then it contains an if to handle them in a different way?

Reply:
becaise there os a lot of repeated code. The other way would be to create helpr functions. @lucasmenendez do you think it makes more sense?

Reply:
Probably creating some helpers could be the way, at least for these operations, which are common to all events:

```go
customer, subscription, err := a.stripe.GetInfoFromEvent(*event)
if err != nil {
	log.Errorf("stripe webhook: error getting info from event: %s\n", err.Error())
	w.WriteHeader(http.StatusBadRequest)
	return
}
address := subscription.Metadata["address"]
if len(address) == 0 {
	log.Errorf("subscription %s does not contain an address in metadata", subscription.ID)
	w.WriteHeader(http.StatusBadRequest)
	return
}
org, _, err := a.db.Organization(address, false)
if err != nil || org == nil {
	log.Errorf("could not update subscription %s, a corresponding organization with address %s was not found.",
		subscription.ID, address)
	log.Errorf("please do manually for creator %s \n  Error:  %s", customer.Email, err.Error())
	w.WriteHeader(http.StatusBadRequest)
	return
}
```